### PR TITLE
[res] TensorFlowLiteRecipes for horizontal FC pattern fusion

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This commit adds models for testing horizontal FC pattern fusion.

for issue: https://github.com/Samsung/ONE/issues/11705
from draft: https://github.com/Samsung/ONE/pull/11739

ONE-DCO-1.0-Signed-off-by: Artem Balyshev <a.balyshev@samsung.com>

### CODE REVIEW

File: res/TensorFlowLiteRecipes/Net_Horizontal_FullyConnected_Add_000/test.rule [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# To check if horizontal FullyConnected and Add kernels are fused
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "ADD_FUSED"               $(op_count ADD) '=' 0
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Net_Horizontal_FullyConnected_Add_000/test.rule:5]: `ADD_REMOVED` ?

Comment (seanshpark) [reply res/TensorFlowLiteRecipes/Net_Horizontal_FullyConnected_Add_000/test.rule:5]: @jinevening , can I get your opinion ?

Comment (jinevening) [reply res/TensorFlowLiteRecipes/Net_Horizontal_FullyConnected_Add_000/test.rule:5]: ADD_REMOVED looks good to me.

Comment (BalyshevArtem) [reply res/TensorFlowLiteRecipes/Net_Horizontal_FullyConnected_Add_000/test.rule]: Done

File: res/TensorFlowLiteRecipes/Net_Horizontal_FullyConnected_Add_000/test.rule [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# To check if horizontal FullyConnected and Add kernels are fused
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "ADD_REMOVED"               $(op_count ADD) '=' 0
+RULE    "FC_FUSED"                $(op_count FULLY_CONNECTED) '=' 1
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Net_Horizontal_FullyConnected_Add_000/test.rule:6]:
```suggestion
RULE    "ADD_REMOVED"             $(op_count ADD) '=' 0
RULE    "FC_FUSED"                $(op_count FULLY_CONNECTED) '=' 1
```

minor change: align

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**